### ISSUE TITLE
there is no metrics  column  on   wandb? only  visual System info

### ISSUE THREAD

**Issue by cqray1990** [ultralytics/yolov5#6623]
![Screenshot 2022-02-13 at 09-42-29 Weights Biases](https://user-images.githubusercontent.com/32585434/153734705-8a13d732-26dd-402a-bc5e-2215f601d5a3.png)
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

when i use wandb show trainin process,but therre is no metrics  column  on   wandb? and at left column  Runs Name is not exp
![Screenshot 2022-02-12 at 19-30-29 Weights Biases](https://user-images.githubusercontent.com/32585434/153710005-bf3381a2-5cb1-4f02-9849-c6f7c0b3a5c1.png)

![Screenshot 2022-02-13 at 09-42-29 Weights Biases](https://user-images.githubusercontent.com/32585434/153734730-03b28e30-e065-4214-b907-c3d2fc87db46.png)

the wandb visualise is not curve,just an point

**Comment by morganmcg1** [comment]
@cqray1990 can you provide a minimal reproducible example?

If only 1 datapoint is logged it will appear as a point instead of a line plot. Maybe try training for more steps? 

Might it be a connection issue? If your network connection dropped then the results mightn't have been pushed to W&B. You can re-sync your wandb logs using ['wandb sync'](https://docs.wandb.ai/ref/cli/wandb-sync) and passing it the path of the logs for that specific wandb run

**Comment by cqray1990** [comment]
before i run train.py,i first run wandb login and enter the keys,then run train.py,as the logging links show,  i visit wandb.ai to see the metric!
@morganmcg1  there nothing changed,

 parser = argparse.ArgumentParser()
    parser.add_argument('--weights', type=str, default=ROOT / 'yolov5x.pt', help='initial weights path')
    parser.add_argument('--cfg', type=str, default=ROOT / 'models/yolov5x.yaml', help='model.yaml path')
    parser.add_argument('--data', type=str, default=ROOT / 'data/tabledet.yaml', help='dataset.yaml path')
    parser.add_argument('--hyp', type=str, default=ROOT / 'data/hyps/hyp.scratch.yaml', help='hyperparameters path')
    parser.add_argument('--epochs', type=int, default=300)
    parser.add_argument('--batch-size', type=int, default=8, help='total batch size for all GPUs, -1 for autobatch')
    parser.add_argument('--imgsz', '--img', '--img-size', type=int, default=640, help='train, val image size (pixels)')
    parser.add_argument('--rect', action='store_true', help='rectangular training')
    parser.add_argument('--resume', nargs='?', const=True, default=True, help='resume most recent training')
    parser.add_argument('--nosave', action='store_true', help='only save final checkpoint')
    parser.add_argument('--noval', action='store_true', help='only validate final epoch')
    parser.add_argument('--noautoanchor', action='store_true', help='disable AutoAnchor')
    parser.add_argument('--evolve', type=int, nargs='?', const=300, help='evolve hyperparameters for x generations')
    parser.add_argument('--bucket', type=str, default='', help='gsutil bucket')
    parser.add_argument('--cache', type=str, nargs='?', const='ram', help='--cache images in "ram" (default) or "disk"')
    parser.add_argument('--image-weights', action='store_true', help='use weighted image selection for training')
    parser.add_argument('--device', default='0', help='cuda device, i.e. 0 or 0,1,2,3 or cpu')
    parser.add_argument('--multi-scale', action='store_true', help='vary img-size +/- 50%%')
    parser.add_argument('--single-cls', action='store_true', help='train multi-class data as single-class')
    parser.add_argument('--optimizer', type=str, choices=['SGD', 'Adam', 'AdamW'], default='SGD', help='optimizer')
    parser.add_argument('--sync-bn', action='store_true', help='use SyncBatchNorm, only available in DDP mode')
    parser.add_argument('--workers', type=int, default=8, help='max dataloader workers (per RANK in DDP mode)')
    parser.add_argument('--project', default=ROOT / 'runs/train', help='save to project/name')
    parser.add_argument('--name', default='exp', help='save to project/name')
    parser.add_argument('--exist-ok', action='store_true', help='existing project/name ok, do not increment')
    parser.add_argument('--quad', action='store_true', help='quad dataloader')
    parser.add_argument('--linear-lr', action='store_true', help='linear LR')
    parser.add_argument('--label-smoothing', type=float, default=0.0, help='Label smoothing epsilon')
    parser.add_argument('--patience', type=int, default=100, help='EarlyStopping patience (epochs without improvement)')
    parser.add_argument('--freeze', nargs='+', type=int, default=[0], help='Freeze layers: backbone=10, first3=0 1 2')
    parser.add_argument('--save-period', type=int, default=1, help='Save checkpoint every x epochs (disabled if < 1)')
    parser.add_argument('--local_rank', type=int, default=-1, help='DDP parameter, do not modify')

    # Weights & Biases arguments
    parser.add_argument('--entity', default=None, help='W&B: Entity')
    parser.add_argument('--upload_dataset', nargs='?', const=True, default=False, help='W&B: Upload data, "val" option')
    parser.add_argument('--bbox_interval', type=int, default=-1, help='W&B: Set bounding-box image logging interval')
    parser.add_argument('--artifact_alias', type=str, default='latest', help='W&B: Version of dataset artifact to use')

    opt = parser.parse_known_args()[0] if known else parser.parse_args()
    return opt
